### PR TITLE
add quickstart v0.1

### PR DESCRIPTION
**Pull request by drrt**

### CODE REVIEW

File: content/spire/try/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+* **Attest a spire-agent to the server using a join token**
+* **Configure a registration entry for a workload**
+* **Fetch an SVID over the WorkloadAPI**
+* **Learn where to find resources for more complex installations on Linux and Kubernetes** 
+
+## Prerequisites
+
+* **A Linux or macOS environment**
+* **The openssl command line tool**
+* **For macOS, Go 1.11 or higher must be installed to build SPIRE. See https://golang.org/dl/ or run `brew install golang`**
```
Comment (ajessup) [inline content/spire/try/quickstart.md]: Is it necessary to build SPIRE for this tutorial? If not, suggest we skip.

File: content/spire/try/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+* **Start the spire-server process**
+* **Attest a spire-agent to the server using a join token**
+* **Configure a registration entry for a workload**
+* **Fetch an SVID over the WorkloadAPI**
+* **Learn where to find resources for more complex installations on Linux and Kubernetes** 
+
+## Prerequisites
+
+* **A Linux or macOS environment**
+* **The openssl command line tool**
```
Comment (ajessup) [inline content/spire/try/quickstart.md:22]: From where should one obtain this tool if it doesn't come with my distro?

File: content/spire/try/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+* **For macOS, Go 1.11 or higher must be installed to build SPIRE. See https://golang.org/dl/ or run `brew install golang`**
+
+The commands in this getting started guide can be run as a standard user or root.
```
Comment (ajessup) [inline content/spire/try/quickstart.md:25]: If it doesn't make a difference, is there value in mentioning this?

File: content/spire/try/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+TTL           : 3600
+Selector      : unix:uid:501
+```
+
+{{< info >}}
+`unix` is just one of the available workload attestation methods. To see a complete list of available attestors, click [here](https://spiffe.io/spire/overview/#selectors).
+{{< /info >}}
+
+
+## Fetch and view a x509 SVID
```
Comment (ajessup) [inline content/spire/try/quickstart.md]: nit, this should be X.509-SVID

**Review by ajessup — COMMENTED** [review]
This looks great! Added some minor comments.

File: content/spire/try/quickstart.md (hunk 2 above)
Comment (drrt) [reply content/spire/try/quickstart.md:22]: It almost certainly is already installed on any Linux or macOS

File: content/spire/try/quickstart.md (hunk 3 above)
Comment (drrt) [reply content/spire/try/quickstart.md:25]: Mostly to point out that you do _not_ need to be root for this quickstart

**Pull request merged**